src/xcode/ENA/ENA/Source/Scenes/ExposureSubmission/View/Controller/ExposureSubmissionTestResultViewModel.swift

```diff
@@ -6,47 +6,49 @@ import UIKit
 import Combine
 
 class ExposureSubmissionTestResultViewModel {
-
+	
 	// MARK: - Init
-
+	
 	init(
 		warnOthersReminder: WarnOthersRemindable,
 		testResult: TestResult,
 		exposureSubmissionService: ExposureSubmissionService,
 		onContinueWithSymptomsFlowButtonTap: @escaping (@escaping (Bool) -> Void) -> Void,
 		onContinueWithoutSymptomsFlowButtonTap: @escaping (@escaping (Bool) -> Void) -> Void,
-		onTestDeleted: @escaping () -> Void
+		onTestDeleted: @escaping () -> Void,
+		onSubmissionConsentButtonTap: @escaping (@escaping (Bool) -> Void) -> Void
```

Comment:
escaping closure use escaping closures as a parameter?
This looks strange to me and might indicate a problem. We should discuss if there is a better solution to do this.

Reply:
As we discussed and agreed, we will think it later on.